torch/nn/modules/module.py

```diff
@@ -47,7 +50,10 @@
 # of `T` to annotate `self`. Many methods of `Module` return `self` and we want those return values to be
 # the type of the subclass, not the looser type of `Module`.
 T = TypeVar("T", bound="Module")
-
+_P = ParamSpec('_P')
```

Comment:
_R isn't used at all either?

Reply:
Forgot to remove it, was changing return types before that and forgot to remove it